modules/bitgo/test/v2/unit/internal/tssUtils/eddsa.ts

```diff
@@ -565,18 +668,52 @@ describe('TSS Utils:', async function () {
         {
           from: 'bitgo',
           to: 'user',
-          publicShare: bitgoKeyShare.yShares[1].y + bitgoKeyShare.yShares[1].v + bitgoKeyShare.yShares[1].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[1].y + params.bitgoKeyShare.yShares[1].v + params.bitgoKeyShare.yShares[1].chaincode,
           privateShare: encryptedBitgoToUserMessage.toString(),
         },
         {
           from: 'bitgo',
           to: 'backup',
-          publicShare: bitgoKeyShare.yShares[2].y + bitgoKeyShare.yShares[2].v + bitgoKeyShare.yShares[2].chaincode,
+          publicShare: params.bitgoKeyShare.yShares[2].y + params.bitgoKeyShare.yShares[2].v + params.bitgoKeyShare.yShares[2].chaincode,
           privateShare: encryptedBitgoToBackupMessage.toString(),
         },
       ],
     };
 
+    const userKeyId = userGpgKeyActual.keyPacket.getFingerprint().padStart(40, '0');
```

Comment:
why the use of padding?  we don't currently do that on the HSM side of things, so this may cause mis-alignment?

Reply:
I was following the example from the OVC:
https://github.com/BitGo/offline-vault-console/blob/master/frontend/src/pkg/bitgo/mpc-utils.ts#L46
I will remove the padding from here to match what happens in the HSM.